src/application/controller/gain_scheduling.h

```diff
@@ -0,0 +1,17 @@
+#ifndef GAIN_SCHEDULING_H_
+#define GAIN_SCHEDULING_H_
+
+#define GAIN_NUM 4
+#define FEEDBACK_GAIN_NUM 3
```

Comment:
```suggestion
#define FEEDBACK_GAIN_NUM (GAIN_NUM-1)
```
shouldn't have multiple independent lengths for defining a union